TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -426,26 +479,47 @@ private void startProxy(BaseProxyObject proxy) {
                 if (getScreenVersion() >= 40602) {
                     logString = " -L -Logfile " + logFile.getAbsolutePath();
                 }
-
-                Process p = new ProcessBuilder(
-                        "/bin/sh", "-c",
-                        "screen -mdS " + proxy.getId() +
-                                logString +
-                                " /bin/sh -c '" +
-                                "cd " + temporaryDirectory.getAbsolutePath() + " &&" +
-                                " " + proxy.getJrePath() + " -server" +
-                                " -Xmx" + proxy.getRam() + "M " +
-                                buildStartParameters(proxy.getJavaParameters()) +
-                                " -Dcom.mojang.eula.agree=true" +
-                                " -Dtimocloud-proxyname=" + proxy.getName() +
-                                " -Dtimocloud-proxyid=" + proxy.getId() +
-                                " -Dtimocloud-corehost=" + TimoCloudBase.getInstance().getCoreSocketIP() + ":" + TimoCloudBase.getInstance().getCoreSocketPort() +
-                                " -Dtimocloud-static=" + proxy.isStatic() +
-                                " -Dtimocloud-templatedirectory=" + templateDirectory.getAbsolutePath() +
-                                " -Dtimocloud-temporarydirectory=" + temporaryDirectory.getAbsolutePath() +
-                                " -jar BungeeCord.jar" +
-                                "'"
-                ).start();
+                if (bungeeJar.exists()) {
```

Comment:
Use auch switch statement or – even better – use the code block and only change the relevant parts of the string.